streaming/src/main/scala/org/apache/carbondata/streaming/parser/FieldConverter.scala

```diff
@@ -50,7 +51,7 @@ object FieldConverter {
       value match {
         case s: String => if (!isVarcharType && !isComplexType &&
                               s.length > CarbonCommonConstants.MAX_CHARS_PER_COLUMN_DEFAULT) {
-          throw new Exception("Dataload failed, String length cannot exceed " +
+          throw new Exception( exceedErrorMsg +
```

Comment:
suggest to use `IllegalArgumentException` can catch it in CarbonScalaUtil.scala

Reply:
done